src/views/assets/Cloud/Strategy/StrategyList.vue

```diff
@@ -69,8 +69,15 @@ export default {
       }
     }
   },
+  computed: {
+    iCategory() {
+      return this.$route.query.category || 'host'
+    }
+  },
+  mounted() {
+    this.tableConfig.url = `/api/v1/xpack/cloud/strategies/?category=${this.iCategory}`
+  },
   methods: {}
-
 }
 </script>
 
```

Comment:
The code you shared seems to be missing some essential parts such as import directives and dependencies. For example:

1. `import` should come before its usage.
2. Dependencies might have changed.

To avoid errors, please ensure the imports are correct with latest versions of all required libraries. Additionally, consider adding a comment at the top explaining exactly what each part does to help maintain consistency and improve readability over time. The commented-out line in the last sample should also include these modifications if present.